scipy/stats/tests/test_distributions.py

```diff
@@ -6349,6 +6349,20 @@ def test_fit_min(self):
         ref = np.mean(rvs), stats.skew(rvs)
         assert_allclose(res, ref)
 
+    # reference values were computed via mpmath
+    # from mpmath import mp
+    # def weibull_sf_mpmath(x, c):
+    #     x = mp.mpf(x)
+    #     c = mp.mpf(c)
+    #     return float(mp.exp(-x**c))
+
+    @pytest.mark.parametrize('x, c, ref', [(50, 1, 1.9287498479639178e-22),
+                                           (1000, 0.8,
+                                            8.131269637872743e-110)])
+    def test_sf_isf(self, x, c, ref):
+        assert_allclose(stats.weibull_min.sf(x, c), ref, rtol=1e-14)
+        assert_allclose(stats.weibull_min.isf(ref, c), x, rtol=1e-14)
```

Comment:
The tests for the inverse survival function fail in main.

Reply:
Yup. You'll need to override `weibull_min._sf` with the same idea as you originally had for `dweibull._sf`.

Reply:
Not sure if I understand exactly: weibull's `sf` method is already overwritten and very accurate.